src/main/java/lotto/ui/Input.java

```diff
@@ -0,0 +1,57 @@
+package lotto.ui;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Input {
+
+    private static final String DELIMITER = ", ";
+
+
+    private Input() {
+        throw new UnsupportedOperationException();
+    }
+
+    public static int getMoneyValue() {
+        System.out.println("구입금액을 입력해 주세요.");
+        final String input = Console.readLine();
+        return getAnInt(input);
+    }
+
+    private static int getAnInt(String input) {
+        return Integer.parseInt(input);
```

Comment:
NumberFormatException 에 대한 예외처리가 있으면 좋을것같네요
```suggestion
        try {
            return Integer.parseInt(input);
        } catch (NumberFormatException ne) {
            throw new IllegalArgumentException("[ERROR] 정수를 입력해주세요.");
        }
```